node/src/main/scala/coop/rchain/node/configuration/Configuration.scala

```diff
@@ -102,9 +102,11 @@ object Configuration {
           s"The value 'pos-multi-sig-quorum' should be less or equal the length of 'pos-multi-sig-public-keys' " +
           s"(the actual values are '$posMultiSigQuorum' and '$posMultiSigPublicKeysLength' respectively)"
       )
-      System.exit(1)
     }
 
+    if (nodeConf.casper.genesisBlockData.posVaultPubKey.isEmpty)
+      System.err.println(s"defaults.conf: " + s"The value 'pos-vault-pub-key' is empty")
```

Comment:
This should be a warning message created by Log instance not system print.

This should be done in `setupNodeProgram` function but only when node is starting to create genesis block. Please ask @nzpr or @zsluedem how to do that.
https://github.com/rchain/rchain/blob/f660b5f79449acbb7b1e9bc4f4d892ae3b32cc09/node/src/main/scala/coop/rchain/node/runtime/Setup.scala#L50-L81

The message should be:
`Public key for PoS vault is not set, manual transfer from PoS vault will be disabled (config 'pos-vault-pub-key')`